src/apps/Calculators/AxByEqualsP/util/getAxByScale.js

```diff
@@ -0,0 +1,43 @@
+const getAxByScale = (firstSteps, firstCents, secondSteps, secondCents) => {
+  let stepDifferences = [];
+
+  if (firstSteps > secondSteps) {
+    for (let i = 0; i < firstSteps; i++) {
+      stepDifferences.push(firstCents);
+    }
+    const disperser = Math.ceil(firstSteps / secondSteps);
+    for (let i = disperser; i <= disperser * secondSteps; i += disperser) {
+      stepDifferences.splice([i], 0, secondCents);
+    }
+  }
+
+  if (firstSteps < secondSteps) {
+    for (let i = 0; i < secondSteps; i++) {
+      stepDifferences.push(secondCents);
+    }
+    const disperser = Math.ceil(secondSteps / firstSteps);
+    for (let i = disperser; i <= disperser * firstSteps; i += disperser) {
+      stepDifferences.splice([i], 0, firstCents);
+    }
+  }
+
+  if (firstSteps === secondSteps) {
+    for (let i = 0; i < firstSteps; i++) {
+      stepDifferences.push(firstCents);
+      stepDifferences.push(secondCents);
```

Comment:
`push` takes multiple args

```suggestion
      stepDifferences.push(firstCents, secondCents);
```